### ISSUE TITLE
next_release: stream did not contain valid UTF-8

### ISSUE THREAD

**Issue by arcnmx** [I60R/page#24]
Say I have a git repo in which I changed file encodings, `git diff | less` shows me something like this:
```diff
             _
-            A<92>B
+            A<U+0092>B
             B
```
(with inverted colour around the `<X>` parts to indicate they're escaped)

`git diff | page` (via `next_release`) doesn't like it one bit:

```
             _
Error reading line from stdin: stream did not contain valid UTF-8
```

It's worth noting that page `master` appears to handle this fine, and passes it along to nvim which displays it relatively sanely (similarly to how less does).

**Comment by I60R** [comment]
Fixed on `next_release` or at least it works for me with UTF-16.

Could you verify if your diff also works?

**Comment by I60R** [comment]
I think it's fixed in 3.0 version
